core-ui/src/components/Extensibility/components-form/StringRenderer.js

```diff
@@ -43,11 +44,11 @@ export function StringRenderer({
         });
       }}
       label={tFromStoreKeys(storeKeys, schema)}
-      placeholder={schemaPlaceholder ? tExt(schemaPlaceholder) : placeholder}
```

Comment:
Without it, translations aren't working.
![image](https://user-images.githubusercontent.com/46759954/190154854-4469933f-3f55-4d39-8992-af3623531df6.png)